Source/Core/RequestType.js

```diff
@@ -0,0 +1,19 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    "use strict";
+
+    /**
+     * @private
+     */
+    var RequestType = {
+        TERRAIN : 0,
+        IMAGERY : 1,
+        TILES3D : 2,
```

Comment:
Down the road we'll need a bit more control here, e.g., what if there are two 3D tilesets: one for a city and one for a point cloud.  Cesium won't know the best priority, but the user might want to override it.

Also, what if a tile has a glTF payload, the glTF requests should be `TILES3D`, but we might also want to bucket them based on geometry vs. texture.

No need to change anything now, but we'll need to think these through.